ts/packages/node/src/stores/LaunchDarkly.ts

```diff
@@ -0,0 +1,154 @@
+      headers: { 'Content-Type': 'application/json; domain-model=launchdarkly.semanticpatch' },
+    });
+    return data;
+  }
+
+  private async getEnvironments(): Promise<string[]> {
+    const { data: environments } = await this.client.get(`/projects/${this.projectKey}/environments`);
+    return environments.items.map((env: any) => env.key);
+  }
+
+  private async getFeatureFlag(config: Config) {
+    const { data: featureFlag } = await this.client.get(`/flags/${this.projectKey}/${config.key}`);
+    return featureFlag;
+  }
+
+  private async createEnvironment(config: Config) {
+    await this.client.post(`/projects/${this.projectKey}/environments`, {
+      color: 'FFFFFF', // This field is required for creating an environment
+      key: config.set,
+      name: config.set,
+    });
+    return config.set;
+  }
+
+  private async createFeatureFlag(config: Config) {
+    let onValue: any;
+    let offValue: any;
+    try {
+      onValue = JSON.parse(config.value);
+      if (typeof onValue === 'boolean') {
+        offValue = !onValue;
+      } else {
+        offValue = Number.isNaN(onValue) ? {} : 0;
+      }
+    } catch (e) {
+      onValue = config.value;
+      offValue = '';
+    }
+    const createData = {
+      key: config.key,
+      name: config.key,
+      variations: [{ value: onValue }, { value: offValue }],
+    };
+    const { data: featureFlag } = await this.client.post(`/flags/${this.projectKey}`, createData);
+    return { featureFlag, onValue };
+  }
+
+  private async createVariation(config: Config) {
+    let onValue: any;
+    try {
+      onValue = JSON.parse(config.value);
+    } catch (e) {
+      onValue = config.value;
+    }
+    const createVariationData = {
+      instructions: [{ kind: 'addVariation', value: onValue }],
+    };
+    const data = await this.patchUpdate(config, createVariationData);
+    return data.variations.find((variation: any) => variation.value === onValue);
+  }
+
+  private async updateDefaultFallthroughVariation(config: Config, variationId: string) {
+    const updateFallthroughVariationData = {
+      environmentKey: config.set,
+      instructions: [
+        {
+          kind: 'updateFallthroughVariationOrRollout',
+          variationId,
+        },
+      ],
+    };
+    return this.patchUpdate(config, updateFallthroughVariationData);
+  }
+
+  private async getEnvFeatureFlags(env: string, keys: string[]): Promise<Config[]> {
+    const { data: featureFlags } = await this.client.get(`/flags/${this.projectKey}?env=${env}`);
+    return featureFlags.items
+      .filter((featureFlag: any) => keys.includes(featureFlag.key))
+      .map((featureFlag: any) => {
+        const onValue = Object.entries(featureFlag.environments[env]._summary.variations)
+          .filter((value: [string, any]) => value[1].isFallthrough)
+          .map((value) => featureFlag.variations[parseInt(value[0], 10)].value)[0];
```

Comment:
Why are we using `_summary` instead of `fallthrough`:
![image](https://github.com/configu/configu/assets/47181182/b5d8314d-4bfd-4166-830c-4e1c3aad8aea)

Reply:
cool. done.
I'm sure you tested this change on every scenario before suggesting it.